tooling/lib/npm.js

```diff
@@ -48,5 +49,8 @@ exports.getDistTag = async function getDistTag(packageName) {
       debug('Something went wrong, but we had to use --silent, so it\'s hard to tell what', err);
     }
   }
+  else if (includeSamples && packageName === 'samples') {
+    return pkg.version;
```

Comment:
Do you need to do the fancy tag check is just returning the package.json sufficient? Does samples need to be versioned if it's private?

Reply:
Samples definitely gets version bumped so we should check it. Not sure why we are going through npm to get the version instead of just package.json